### PR TITLE
docs/ipsec: Extend troubleshooting for long key rotations

### PR DESCRIPTION
**Pull request by pchaigno**
This is documentation for the flag introduced in https://github.com/cilium/cilium/pull/24811 and the problem it addresses.

### CODE REVIEW

**Review by gentoo-root — APPROVED** [review]

File: Documentation/security/network/encryption-ipsec.rst [hunk 1]
```diff
@@ -211,15 +211,25 @@ Troubleshooting
...
 
+ * After a key rotation, if the old key is cleaned up before the
+   configuration of the new key is installed on all nodes, it will result in
+   ``XfrmInNoStates`` errors. The old key is removed from nodes after 5 minutes
+   by default. By default, all agents watch for key updates and update their
+   configuration within 1 minute after the key is changed, leaving plenty of
+   time before the old key is removed. If you expect the key rotation to take
+   longer for some reason (e.g., in case of clustermesh several clusters need
+   to be updated), you can increase the delay before cleanup with agent flag
+   ``ipsec-key-rotation-duration``.
```
Comment (zacharysarah) [inline Documentation/security/network/encryption-ipsec.rst]: Edits for clarity, future tense, no Latin abbreviations. Normally I would review for passive voice, but I think active voice alternatives wouldn't be nearly as clear.
```suggestion
   configuration of the new key is installed on all nodes, it results in
   ``XfrmInNoStates`` errors. The old key is removed from nodes after a default interval
   of 5 minutes. By default, all agents watch for key updates and update their
   configuration within 1 minute after the key is changed, leaving plenty of
   time before the old key is removed. If you expect the key rotation to take
   longer for some reason (for example, in the case of Cluster Mesh if several clusters need
   to be updated), you can increase the delay before cleanup with agent flag
   ``ipsec-key-rotation-duration``.
```

**Review by zacharysarah — CHANGES_REQUESTED** [review]
@pchaigno Some mild edits for the usual suspects

File: Documentation/security/network/encryption-ipsec.rst (hunk 1 above)
Comment (pchaigno) [reply Documentation/security/network/encryption-ipsec.rst]: > future tense

I really need to be more careful on that. I know it and you've pointed it before :disappointed: 

> no Latin abbreviations

FYI, regarding that and "clustermesh" vs. "Cluster Mesh", the docs are not consistent (even this page isn't).

**Review by zacharysarah — APPROVED** [review]
@pchaigno Thanks for the updates!

**Pull request merged**